products/jbrowse-web/src/SessionLoader.ts

```diff
@@ -195,12 +191,13 @@ const SessionLoader = types
      * #getter
      */
     get isSessionLoaded() {
-      return Boolean(
-        self.sessionError ||
-          self.sessionSnapshot ||
-          self.blankSession ||
-          self.sessionSpec,
-      )
+      if (self.sessionError || self.blankSession || self.sessionSpec) {
+        return true
+      }
+      if (self.sessionSnapshot && self.sessionPlugins) {
```

Comment:
can this be keept the same as before and then add above diff to above get ready

Reply:
Restored